crates/miden-lib/src/account/auth/mod.rs

```diff
@@ -207,6 +211,43 @@ impl From<AuthRpoFalcon512Acl> for AccountComponent {
     }
 }
 
+/// An [`AccountComponent`] implementing a no-authentication scheme.
+///
+/// This component provides minimal authentication by only checking if the account
+/// state has actually changed during transaction execution. It compares the initial
+/// account commitment with the current commitment and only increments the nonce if
+/// they differ, avoiding unnecessary nonce increments for transactions that don't
+/// modify the account state.
```

Comment:
```suggestion
/// This component provides **no authentication**! If only checks if the account
/// state has actually changed during transaction execution by comparing the initial
/// account commitment with the current commitment and increments the nonce if
/// they differ. That avoids unnecessary nonce increments for transactions that don't
/// modify the account state.
```

Nit: I'd make this statement stronger to really drive the point home that this does absolutely no authentication.